dspace/config/modules/bitstore-srb.cfg

```diff
@@ -0,0 +1,57 @@
+##### SRB File Storage #####
```

Comment:
It looks as though the SRB (Storage Resource Broker) code is all removed in this PR, which makes me wonder why this "bitstore-srb.cfg" file is even needed?

This PR should likely also mention that it removes the outdated SRB code (which hasn't been maintained for some time).